docs/about.md

```diff
@@ -0,0 +1,80 @@
+# About Heptio Ark
+
+Heptio Ark provides customizable degrees of recovery for all Kubernetes objects (Pods, Deployments, Jobs, Custom Resource Definitions, etc.), as well as for persistent volumes. This recovery can be cluster-wide, or fine-tuned according to object type, namespace, or labels.
+
+Ark is ideal for the disaster recovery use case, as well as for snapshotting your application state, prior to performing system operations on your cluster (e.g. upgrades).
+
+## Features
+
+Ark provides the following operations:
+
+* On-demand backups
+* Scheduled backups
+* Restores
+
+Each operation is a custom resource, defined with a Kubernetes [Custom Resource Definition (CRD)][20] and stored in [etcd][22]. An additional custom resource, Config, specifies cloud provider settings. These resources are handled by [custom controllers][21] when their corresponding requests are submitted to the Kubernetes API server.
```

Comment:
I don't think `Config` is solely about cloud provider settings, though those are certainly included. Maybe something like "An additional custom resource, Config, specifies required information and customization options, such as cloud provider settings".

Reply:
I'm fine if we leave this as-is, or if we make changes. Not a huge deal, especially once we get to backup targets.